src/main/java/data/AnalysisResults.java

```diff
@@ -0,0 +1,26 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+/**
+ * Object for holding the analysis results served by ReceiptAnalysisServlet.
+ */
+public class AnalysisResults {
+  private String rawText;
```

Comment:
Always mark an instance variable (field) final if it is never reassigned (e.g., private final String rawText;). See Effective Java Item 17 "Minimize mutability".